src/dataflow-types/src/types.rs

```diff
@@ -145,11 +145,13 @@ pub struct DataflowDescription<View, T = mz_repr::Timestamp> {
     pub debug_name: String,
     /// Unique ID of the dataflow
     pub id: GlobalId,
+    /// The target compute instance for this dataflow.
+    pub compute_instance_id: usize,
```

Comment:
Is this necessary? The `create_dataflows` command takes a compute instance id as an argument.